tools/config_updater.py

```diff
@@ -479,9 +510,7 @@ def Initialize(config=None, token=None):
   startup.ConfigInit()
 
   print "\nStep 3: Adding Admin User"
-  password = getpass.getpass(prompt="Please enter password for user 'admin': ")
-  UpdateUser("admin", password, ["admin"], token=token)
-  print "User admin added."
+  AddUser("admin", ["admin"], token=token)
```

Comment:
This sets password to ["admin"]. Please call named arguments by name (labels=["admin"]) and I'd also suggest you add a default of None to the password parameter so you don't have to do AddUser("admin", None, labels=...)